src/sage/doctest/external.py

```diff
@@ -398,6 +398,7 @@ class AvailableSoftware():
          'magma',
          'maple',
          'mathematica',
+         'mathics',
```

Comment:
Why this? Mathics is below `$SAGE_ROOT` and has a pinned version in `build/pkgs`. Can't we consider Mathics as a Python-Feature instead of an Interface-Feature (or do something else to keep it out of `external_features`)? I think it cannot be wrong if it would be tested on every system where the optional package is available.

Reply:
Done in 4bf041bc0c7

Reply:
Thanks!